osu.Game/Online/API/Requests/Responses/APISystemTitle.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Newtonsoft.Json;
+
+namespace osu.Game.Online.API.Requests.Responses
+{
+    public record APISystemTitle
```

Comment:
`record` is for free deep equality. Without it, reference equality is used in `SystemTitle`, causing the same banner to be unloaded and reloaded again.

Reply:
nvm https://github.com/ppy/osu/pull/26172/commits/ac449131edcee95d9ca61fef538aeea7010c128d